test/getCoverageLIb.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, it } from "node:test";
+import assert from "node:assert";
+import { getCoverageLib } from "../src/utils";
+
+describe("getCoverageLib.js", () => {
+  it("getCoverageLib should return N/A", () => {
+    assert.strictEqual(getCoverageLib({ test: "2.0.0" }), "N/A");
+  });
+
+  it("getCoverageLib should return nyc", () => {
+    assert.strictEqual(getCoverageLib({ nyc: "2.0.0" }), "nyc");
+  });
+  it("getCoverageLib should return c8", () => {
+    assert.strictEqual(getCoverageLib({ c8: "2.0.0" }), "c8");
+  });
```

Comment:
```suggestion
  it("getCoverageLib should return nyc", () => {
    assert.strictEqual(getCoverageLib({ nyc: "2.0.0" }), "nyc");
  });

  it("getCoverageLib should return c8", () => {
    assert.strictEqual(getCoverageLib({ c8: "2.0.0" }), "c8");
  });
```
Can you please skip line between each tests ?